app/src/pages/how-it-works.tsx

```diff
@@ -28,7 +31,7 @@ const HowItWorks: NextPage = () => {
         ))}
       </ol>
       <Alert alertBody="HowItWorks.note" type="warning" />
-      <ButtonLink href="/eligibility" labelKey="HowItWorks.button" />
+      <ButtonLink href={forwardRoute} labelKey="HowItWorks.button" />
     </>
   )
 }
```

Comment:
Notes for reviewers: The HowItWorks page (`/how-it-works`) is the second page for the eligibility screener. It is a content-only page. The back link routes to `/` and the call to action button routes the user to `/eligibility`. There are no data validation guards for this page because it is a content-only page in the form wizard flow.

![Screen Shot 2022-10-28 at 1 40 43 AM](https://user-images.githubusercontent.com/67701/198544274-c4cc21f2-aea2-467a-a492-a606bcb7f0af.png)